script/test/utils.sh

```diff
@@ -166,12 +166,15 @@ version = 2
           SandboxIsolation = 1
 EOF
 fi
-
-if [ $IS_WINDOWS -eq 0 ] && [ ! -z "$CGROUP_DRIVER" ] && [ "$CGROUP_DRIVER" = "systemd" ];then
+# To allow the cri-integration test to run via CLI without explicitly setting CGROUP_DRIVER
+if [ $IS_WINDOWS -eq 0 ] && [ ! -v CGROUP_DRIVER ]; then
+  echo “CGROUP_DRIVER is unset”
+  else if [ “$CGROUP_DRIVER” = “systemd” ];then
```

Comment:
The bash syntax would be `elif`.

Reply:
here what we are doing:

```
if condition; then
  operation
else
  if condition; then
    operation
  fi
fi
```
can also be done in short hand
```
if condition; then
  operation
elif condition; then
  operation
fi
```
both are correct, IMO

Reply:
Oh I see now that's why the second fi. Yeah it's functional 👍